first-step-spring-class-config/src/main/java/org/elearning/portfolio/services/UserService.java

```diff
@@ -0,0 +1,48 @@
+package org.elearning.portfolio.services;
+
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import org.elearning.portfolio.user.*;
+import org.elearning.portfolio.message.*;
+
+
+public class UserService{
+
+    @Autowired
+    private  UserDAO user  ;
```

Comment:
Podobnie jak w przypadku pola "message" tutaj nazwij to userDao, bo user kojarzy się z encją, DTO lub VO (value object)